stable/testlink/values.yaml

```diff
@@ -243,8 +243,14 @@ ingress:
   ## It is also possible to create and manage the certificates outside of this helm chart
   ## Please see README.md for more information
   # - name: testlink.local-tls
-  #   key:
-  #   certificate:
+  #   key: |-
+  #     -----BEGIN PRIVATE KEY-----
```

Comment:
Why ` -----BEGIN PRIVATE KEY-----` and not `-----BEGIN RSA PRIVATE KEY-----` ?

Reply:
They are just different formats: https://tls.mbed.org/kb/cryptography/asn1-key-structures-in-der-and-pem
Personaly, I've never seen a `-----BEGIN RSA PRIVATE KEY-----`, all the certificates I ever bought were always `-----BEGIN PRIVATE KEY-----`, but no problem, I'll keep this comment like the others